dashboard/app/views/pd/workshop_mailer/_covid_19.html.haml

```diff
@@ -0,0 +1,32 @@
+%h3 COVID-19 precautions
+
+%p
+  In light of the global situation surrounding the COVID-19 virus, we understand that traveling and
+  gathering for a workshop may be cause for anxiety or concern for teachers registered for upcoming
+  Code.org workshops. We encourage you to refer to the
+  = link_to 'Centers for Disease Control and Prevention', 'https://www.cdc.gov/', target: '_blank'
+  (CDC) for the most up to date recommendations, as is your Regional Partner in evaluating whether
+  to host your workshops as planned. If you have returned from a
+  = link_to 'Level 3 country', 'https://www.cdc.gov/coronavirus/2019-ncov/travelers/index.html', target: '_blank'
+  within the past 14-days, are feeling ill or show any flu-like symptoms such as a fever or cough,
+  please inform your Regional Partner.
+
+%p
+  Our workshops are highly interactive. At time of this update, the CDC suggests all individuals
+  = link_to 'follow this guidance', 'https://www.cdc.gov/coronavirus/2019-ncov/about/prevention-treatment.html', target: '_blank'
+  during flu season and to prevent the spread of the COVID-19 virus, including:
+  %ul
+    %li
+      Avoid contact with sick people.
+    %li
+      Avoid touching your eyes, nose, or mouth with unwashed hands.
+    %li
+      Clean your hands often by washing them with soap and water for at least 20 seconds or using
+      an alcohol-based hand sanitizer that contains at least 60% alcohol.
+    %li
+      It is especially important to clean hands after going to the bathroom; before eating; and
```

Comment:
If we're using semicolons, suggest:

It is especially important to clean hands: after going to the bathroom; before eating; and after coughing, sneezing, or blowing your nose.